### PR TITLE
Implement announcements

### PR DESCRIPTION
**Pull request by spd75**
## Overview
Implemented announcements in Volume.

## Changes Made
Connected with AppDevAnnouncements pod. 

Secrets.plist now has announcements values.

I also changed the Readme.md because downloading the schema.json has never once worked for me.  Plus, this requires backend developers to send a new schema.json every time they change the schema.  This is misleading and isn't sustainable, so I updated the Readme.md with the method I use to get the schema.json.

## Test Coverage
Displayed multiple test announcements.

## Screenshots (delete if not applicable)

<details>

  <summary>Final Test Screenshot</summary>

Screen Shot 2021-03-22 at 11 56 26 AM
  

</details>

### CODE REVIEW

File: Volume/Analytics/Analytics.swift [hunk 1]
```diff
@@ -9,8 +9,38 @@
...
+    var parameters: [String: Any]? {
+        nil
+    }
+}
+
+struct AnnouncementPresentedPayload: Payload {
+    let eventName = "announcement_presented"
+}
+
+class VolumeAppDevAnalytics {
```
Comment (danielVebman) [inline Volume/Analytics/Analytics.swift:13]: Remove new lines on 13 and 40

**Review by danielVebman — APPROVED** [review]
Lgtm 🚀

File: Volume/Analytics/Analytics.swift (hunk 1 above)
Comment (spd75) [reply Volume/Analytics/Analytics.swift:13]: Line 13 is `var eventName: String { get }`
Do you mean line 11?

Comment (danielVebman) [reply Volume/Analytics/Analytics.swift:13]: 11 28 and 40. Idk where I got 13 from.

**Review by camruss — APPROVED** [review]

**Pull request merged**